answers/Maker.md

```diff
@@ -0,0 +1,179 @@
+
+MKR holders are incentivized to vote on proposals that preserve the stability of the system so that a wide range of actors can use the system and generate stability fees, which must be paid in MKR. When a user pays the stability fee in MKR to unlock their collateral, the MKR is burned, decreasing the total supply.
+
+#### Governance votes
+
+Governance votes can be used to vote on one or multiple issues at once. They do not automatically trigger updates to the Maker system. Instead, these must be implemented manually by the Maker team. Nor are they binding resolutions. Rather, they are used to poll community sentiment towards larger, more substantial changes to the system. This can include making changes to the structure or governance processes of the Maker Foundation, including adding new Oracles, adding a new risk team (people that create and apply risk models), or adopting a new voting process. Votes can be time-limited. If the vote is time-limited, votes are tallied at the end of the voting period and a simple majority (<50%) determines the outcome.
+
+#### Executive votes
+
+Executive votes are a more common occurrence, and are used to change the state, or "governance variables" of the smart contracts constituting the Maker infrastructure. Typically this means modifying the existing "risk parameters" of smart contracts that manage Collateral Debt Positions (CDPs), the debt instruments used to issue DAI and manage its supply. For example, an executive vote could be held to decide whether or not to raise the "stability fee" (i.e. "interest" paid to MKR holders on loans). Executive votes can also introduce new parameters or smart contracts. For example adding a new collateral type once multi-collateral DAI is launched.
+
+Executive votes are binding. If passed, they are automatically implemented on the blockchain after a 24 hr delay (a measure to protect against hacks or governance attacks). However, in practice, while any Ethereum address can make a proposal and trigger a vote, the concentration of MKR holdings in the Maker Foundation, core team, and associated investors (Polychain, Andreessen Horowitz, Placeholder VC, among others), mean that only executive votes created by the core team have a reasonable chance of passing. For now, proposals for executive votes are created in a more traditional, centralized process within the Maker Foundation, utilizing a formal process that attempts to emulate the scientific process. Feedback from MKR holders and the general "governance community" is taken into account at various stages. Maker's goal is to perform a "gradual decentralization" of this process over time as the system matures.
```

Comment:
Needs links, especially the part about concentration of holdings.

Reply:
Added more details about holdings in the 'Coin distribution' section, and reference that from here. Also reworded a bit to try and be as accurate as possible.